agent/src/main/java/reactor/blockhound/BlockingOperationError.java

```diff
@@ -0,0 +1,12 @@
+package reactor.blockhound;
+
+public class BlockingOperationError extends Error {
```

Comment:
Let's add `BlockingMethod`s fields to this error, so that we can use it for asserting that the correct method was detected. This way, we can also move the message creation logic to `toString` of `BlockingOperationError`

Reply:
on a second thought, let's just add `BlockingMethod` field to it, no need to "inline" the fields :)